bundle/Controller/LocationController.php

```diff
@@ -17,9 +19,12 @@ class LocationController extends Controller
 
     protected $configuration;
 
-    public function __construct(LocationService $locationService)
+    protected $logger;
+
+    public function __construct(LocationService $locationService, LoggerInterface $logger)
```

Comment:
The logger should be made optional, as it can be disabled.

I recommend you use `Psr\Log\LoggerAwareTrait`, and replace the `@logger` injection below with a call to `setLogger()` with `@?logger` as the argument. Also make sure that when you use the logger, you take into account the fact that the logger might be `null`.